backend/tests/test_main.py

```
@@ -0,0 +1,91 @@
+@pytest.fixture
+def client():
+    with TestClient(app) as client:
+        yield client
+
+@pytest.fixture(autouse=True)
+def setup_database():
+    Base.metadata.create_all(bind=engine)
+    yield
+    Base.metadata.drop_all(bind=engine)
+
+def test_cors_middleware(client):
```

Ruff (0.8.2)

Missing return type annotation for public function `test_cors_middleware`

Add return type annotation: `None`

(ANN201)